contents/ToC.md

```diff
@@ -27,4 +27,5 @@
 
 ## Other Tools/Technologies
 
-* AI
\ No newline at end of file
+* AI
+* [Regex](regex/Regex.md)
```

Comment:
Make the visible text "Regular Expressions" to cater to those who may not know what "Regex" means.